feat!(nsis): add an option to customize start menu folder (#9994)
tauri-apps/tauri: .changes/nsis-start-menu-folder.md

---
"tauri-utils": "patch:feat"
"tauri-bundler": "patch:breaking"
---

> Add an option to customize start menu folder for NSIS installer, and changed default to no additional folder instead of an extra folder with the same name as the shortcut

Reviewer: let's separate the breaking change in another change file

> 
> Add an option to customize start menu folder for NSIS installer, and changed default to no additional folder instead of an extra folder with the same name as the shortcut

Reviewer: ```suggestion

Add `bundle > nsis > startMenuFolder` option to customize start menu folder for NSIS installer
```
